k8s/common/docmosis.yaml

```diff
@@ -1,5 +0,0 @@
----
```

Comment:
:+1:

Reply:
Branch now rolled back to previous approved commit.